controllers/search.js

```diff
@@ -0,0 +1,175 @@
+import Sequelize from 'sequelize';
+import models from '../models';
+
+const { User, article } = models;
+
+const userAttrib = ['username', 'image', 'bio'];
+const articleAttrib = ['title', 'description', 'slug', 'tagList'];
+
+const handler = {
+  searchByAuthor: async (req, res) => {
+    const { title, username } = req.body;
+
+    if (!title && !username) {
```

Comment:
Consider trimming the variables, because when the title is a single space(which passes this _null_ test) I get all articles!